.github/workflows/release.yaml

```diff
@@ -96,12 +96,12 @@ jobs:
           images: ${{ env.IMAGE }}
           flavor: |
             latest=false
-            - name: Load Secrets from Vault
-              uses: rancher-eio/read-vault-secrets@main
-              with:
-                secrets: |
-                  secret/data/github/repo/${{ github.repository }}/dockerhub/rancher/credentials username | DOCKER_USERNAME ;
-                  secret/data/github/repo/${{ github.repository }}/dockerhub/rancher/credentials password | DOCKER_PASSWORD
+      - name: Load Secrets from Vault
+        uses: rancher-eio/read-vault-secrets@main
+        with:
+          secrets: |
+            secret/data/github/repo/${{ github.repository }}/dockerhub/rancher/credentials username | DOCKER_USERNAME ;
+            secret/data/github/repo/${{ github.repository }}/dockerhub/rancher/credentials password | DOCKER_PASSWORD
       - name: Login to Docker Hub
```

Comment:
@jiaqiluo do we need to update this section to read from secrets as well?

Reply:
```
      - name: Login to Docker Hub
        uses: docker/login-action@v3
        with:
          username: ${{ env.DOCKER_USERNAME }}
          password: ${{ env.DOCKER_PASSWORD }}
 ```

Reply:
Based on the eio example, no

```yaml
    - name: "Read some Secrets"
      uses: rancher-eio/read-vault-secrets@main
      with:
        secrets: |
          secret/data/github/repo/${{ github.repository }}/dockerhub/credentials username | DOCKER_USERNAME ;
          secret/data/github/repo/${{ github.repository }}/dockerhub/credentials password | DOCKER_PASSWORD
    - name: "Use those Secrets"
      run: |
        echo "${{ env.DOCKER_USERNAME }}
```